modules/hub/pages/autoconnect-mode.adoc

```diff
@@ -0,0 +1,53 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+The autoconnect mode is a convenient method where the same credentials used to login into the Hub API are used to login into the APIs of the servers the user wants to work with.
+It is assumed the credentials are valid in each server and correspond to an admin user. Difference from relay mode is that there is no need to call `attachToServers` method which also means that there is no need to call `listServerIds` method to get the `serverIds`.
```

Comment:
```suggestion
Differently from relay mode, there is no need to use the `attachToServers` method, as it is assumed the user wants to connect to all peripheral Servers under his control, and the Hub XMLRPC API will in fact log in to them all at Hub login time.
```

Reply:
while you are at it, can we change every occurrence of "serverIds" or derivative forms to "serverIDs" ? (Including example scripts)